File: 0168.excel_sheet_column_title/solution.java
class Solution {
    public String convertToTitle(int n) {
        StringBuffer ans = new StringBuffer();
        while (n>0){
            int mod = n % 26;
            n = n / 26;
            if (mod == 0){
                mod = 26;
                n--;
            }
            ans.append((char) mod+64);
        }
        return ans.reverse().toString();
    }
}

File: 0168.excel_sheet_column_title/solution.py
class Solution:
    def convert_to_title(self, n: int) -> str:
        """
        1 -> 'A'
        27 -> 'AA'
        703 -> 'AAA'
        """
        ans = []
        while n > 0:
            n, mod = divmod(n, 26)
            if mod == 0:
                mod = 26
                n -= 1
            ans.append(chr(64 + mod))
        return "".join(ans[::-1])
